adapters/handlers/rest/handlers_backup.go

```diff
@@ -91,9 +93,20 @@ func parseCompressionLevel(l string) ubak.CompressionLevel {
 func (s *backupHandlers) createBackup(params backups.BackupsCreateParams,
 	principal *models.Principal,
 ) middleware.Responder {
+	fmt.Printf("BackupHandler: %+v\n", params)
+	fmt.Printf("Body: %+v\n", params.Body)
+	fmt.Printf("Config: %+v\n", params.Body.Config)
+	OverrideBucket := ""
+	OverridePath := ""
```

Comment:
can we make those variables private?

Reply:
It is private, it is not possible to make scoped variables public.

Reply:
my point is can those variables start with lower cased letter?

Reply:
can we make those variables to start with lowercase letter? just to follow the camel case convention